src/exception.cc

```diff
@@ -0,0 +1,11 @@
+#include<micro/core/exception.h>
```

Comment:
This file is very small and maybe is a better idea to remove it and put the code directly into the .h file.